src/axom/spin/tests/spin_bvhtree.cpp

```diff
@@ -5,19 +5,18 @@
 
 #include "gtest/gtest.h"
 
-#include "axom/primal/geometry/BoundingBox.hpp"
-#include "axom/primal/spatial_acceleration/BVHTree.hpp"
-#include "axom/primal/geometry/Point.hpp"
+#include "axom/spin/BVHTree.hpp"
+#include "axom/spin/Primitives.hpp"
 
 using namespace axom;
 
 //------------------------------------------------------------------------------
-TEST( primal_bucket_tree, insert_object )
+TEST( spin_bucket_tree, insert_object )
 {
   // some typedef short-cuts
-  typedef primal::BVHTree< int,3 > TreeType;
-  typedef primal::BoundingBox< double,3 > BoxType;
-  typedef primal::Point< double,3 > PointType;
+  using TreeType = spin::BVHTree< int,3 > ;
+  using BoxType = spin::BoundingBox< double,3 > ;
+  using PointType = spin::Point< double,3 > ;
```

Comment:
Here is another instance: ``BoxType`` is an alias ``spin::BoundingBox``, which is an alias to ``primal::BoundingBox``. This indirection would be confusing. 

For this and in all other similar instances, it would be better if you the alias maps directly to the actual type instead of another alias. For example do this instead:
```
using BoxType = primal::BoundingBox< double, 3 >;
```

Reply:
as before